### PR TITLE
reorgd default scheduler

### PR DESCRIPTION
**Pull request by mburke5678**
Follow-up to https://github.com/openshift/openshift-docs/pull/7313

### CODE REVIEW

File: admin_guide/scheduling/scheduler.adoc [hunk 1]
```diff
@@ -536,27 +540,30 @@ The default scheduler policy includes the priorities noted in the list. Each of
 the priority function has a weight of `1` except `*NodePreferAvoidPodsPriority*`,
 which has a weight of `10000`:
 
+[[default-predicates]]
```
Comment (vikram-redhat) [inline admin_guide/scheduling/scheduler.adoc]: @mburke5678 - duplicate id. Should this have been default-priorities?

**Comment by vikram-redhat** [comment]
@mburke5678 - if this is a follow up to #7313 and 7313 was CP'ed to 3.6, 3.7 and 3.9, should this have been CP'ed to 3.6 and 3.9 as well?

**Comment by mburke5678** [comment]
@vikram Yes. The changes I made in this PR can be applied to 3.6 and 3.9 through a separate PR, as the lists of predicates and priorities are different in each version.

**Comment by mburke5678** [comment]
3.9 https://github.com/openshift/openshift-docs/pull/8210
3.7 https://github.com/openshift/openshift-docs/pull/8087
3.6 https://github.com/openshift/openshift-docs/pull/8211

**Pull request merged**